packages/devcmd/src/from-cli/index.ts

```diff
@@ -1,8 +1,10 @@
 import { promises as fs } from "fs";
+import { green } from "kleur";
 import { gray, bold, red, reset } from "kleur/colors";
 import { spawnSync } from "npm-run";
 import path from "path";
 import { formatCommandArgs, formatCommandName } from "../utils/format_utils";
+import { getReservedCommand, reservedCommands } from "../utils/get_reserved_command";
```

Comment:
is there a reason why we use both getReservedCommand and reservedCommands here?

I would expect the information to be a little more unified (either an exported immutable map of name to command function, or otherwise getReservedCommand being used to look up the requested command)